core/Makefile

```diff
@@ -135,3 +135,28 @@ find-absolute-dlls:
 
 check-copyrights:
 	python ./updateCopyrightCheck.py
+
+eclipse: rest/.project sql/.project sqf/src/seatrans/hbase-trx/.project .project
+	@echo
+	@echo
+	@echo "  Now you can import 5 projects in Eclipse:"
```

Comment:
The HBase-trx make step actually creates two projects, hbase-trx and hbase-protocol.